ddtrace/contrib/redis/patch.py

```diff
@@ -36,14 +37,15 @@ def patch():
         _w("redis", "Redis.pipeline", traced_pipeline)
         _w("redis.client", "Pipeline.execute", traced_execute_pipeline)
         _w("redis.client", "Pipeline.immediate_execute_command", traced_execute_command)
-        if redis.VERSION >= (4, 2, 0):
-            from .asyncio_patch import traced_async_execute_command
-            from .asyncio_patch import traced_async_execute_pipeline
-
-            _w("redis.asyncio.client", "Redis.execute_command", traced_async_execute_command)
-            _w("redis.asyncio.client", "Redis.pipeline", traced_pipeline)
-            _w("redis.asyncio.client", "Pipeline.execute", traced_async_execute_pipeline)
-            _w("redis.asyncio.client", "Pipeline.immediate_execute_command", traced_async_execute_command)
+        if PY3:
```

Comment:
This change might conflict with this PR: https://github.com/DataDog/dd-trace-py/pull/3868. Should we revert this change?